src/main/java/net/pms/util/Constants.java

```diff
@@ -11,16 +11,19 @@
 	public static final String CHARSET_ISO_8859_7	= "ISO-8859-7".intern();
 	public static final String CHARSET_ISO_8859_8	= "ISO-8859-8".intern();
 	public static final String CHARSET_ISO_8859_9	= "ISO-8859-9".intern();
+	public static final String CHARSET_ISO_8859_11	= "ISO-8859-11".intern();
```

Comment:
@valib Snif snif...
Sadly the only Java libraries that i found have czech or thaï but not the both :wink: